src/matcha_ml/cli/cli.py

```diff
@@ -242,5 +248,22 @@ def opt_in() -> None:
     core.analytics_opt_in()
 
 
+@stack_app.command(help="Define the stack for Matcha to provision.")
+def set(stack: str = typer.Argument("default")) -> None:
+    """Define the stack for Matcha to provision.
+
+    Args:
+        stack (str): the name of the stack to provision.
+
+    Raises:
+        Exit: Exit if input is not a defined stack.
+    """
+    try:
+        core.stack_set(stack)
```

Comment:
The return value of `stack_set` isn't captured as a variable which could be used to later to signal to the user that the stack as been set. It's important that the user has confirmation of their actions when interacting via the CLI.

You could use `print_status` in the `ui` module to output the result, after the `try: ... except: ...`

The presence of the stack name passed to the CLI can then be tested for, along with the function call.

Reply:
Added.